src/main/java/plugily/projects/buildbattle/arena/states/guess/InGameState.java

```diff
@@ -248,64 +244,62 @@ private void openThemeSelectionInventoryToCurrentBuilder(GuessArena pluginArena)
       }
     });
-    List<String> themes = pluginArena.getPlugin().getThemeManager().getThemes(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_EASY");
+
     String itemLore = new MessageBuilder("MENU_THEME_GTB_ITEM_LORE").asKey().arena(pluginArena).build();
     String themeItemName = new MessageBuilder("MENU_THEME_GTB_ITEM_NAME").asKey().arena(pluginArena).build();
-    String themeNameEasy = !themes.isEmpty() ? themes.get(themes.size() == 1 ? 0 : random.nextInt(themes.size())) : "";
-    setTheme(pluginArena, themeNameEasy, BBTheme.Difficulty.EASY);
+
+    setTheme(pluginArena, BBTheme.Difficulty.EASY);
+
     gui.setItem(11, new SimpleClickableItem(new ItemBuilder(Material.PAPER).name(themeItemName)
         .lore(itemLore
             .replace("%difficulty%", new MessageBuilder("MENU_THEME_GTB_DIFFICULTIES_EASY").asKey().build())
             .replace("%points%", "1").split(";")).build(), event -> {
 
     }));
 
-
-    themes = pluginArena.getPlugin().getThemeManager().getThemes(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_MEDIUM");
-    String themeNameMedium = !themes.isEmpty() ? themes.get(themes.size() == 1 ? 0 : random.nextInt(themes.size())) : "";
-    setTheme(pluginArena, themeNameMedium, BBTheme.Difficulty.MEDIUM);
+    setTheme(pluginArena, BBTheme.Difficulty.MEDIUM);
     gui.setItem(13, new SimpleClickableItem(new ItemBuilder(Material.PAPER).name(themeItemName)
         .lore(itemLore
             .replace("%difficulty%", new MessageBuilder("MENU_THEME_GTB_DIFFICULTIES_MEDIUM").asKey().build())
             .replace("%points%", "2").split(";")).build(), event -> {
 
     }));
 
-    themes = pluginArena.getPlugin().getThemeManager().getThemes(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_HARD");
-    String themeNameHard = !themes.isEmpty() ? themes.get(themes.size() == 1 ? 0 : random.nextInt(themes.size())) : "";
-    setTheme(pluginArena, themeNameHard, BBTheme.Difficulty.HARD);
+    setTheme(pluginArena, BBTheme.Difficulty.HARD);
+
     gui.setItem(15, new SimpleClickableItem(new ItemBuilder(Material.PAPER).name(themeItemName)
         .lore(itemLore
             .replace("%difficulty%", new MessageBuilder("MENU_THEME_GTB_DIFFICULTIES_HARD").asKey().build())
             .replace("%points%", "3").split(";")).build(), event -> {
 
     }));
-    gui.open(pluginArena.getCurrentBuilder());
 
+    gui.open(pluginArena.getCurrentBuilder());
   }
 
-  private void setTheme(GuessArena pluginArena, String themeName, BBTheme.Difficulty difficulty) {
-    BBTheme theme = new BBTheme(ChatColor.stripColor(themeName), difficulty);
-    pluginArena.setCurrentTheme(theme);
+  private boolean setTheme(GuessArena pluginArena, BBTheme.Difficulty difficulty) {
+    List<String> themes = pluginArena.getPlugin().getThemeManager().getThemes(ThemeManager.GameThemes.getByDifficulty(difficulty));
+    String themeName = !themes.isEmpty() ? themes.get(themes.size() == 1 ? 0 : random.nextInt(themes.size())) : "";
```

Comment:
boolean isNotEmpty can be already used ealier 
renaming it to isThemeListEmpty would make more sense and then using it with ! or without